structure.f

```diff
@@ -307,6 +310,14 @@ subroutine Basis_Builder( system , basis , GACG_flag )
    first_time = .false.
 end do
 
+if( SOC ) then
+
+    k = N_of_orbitals / 2
+
+    basis( k + 1 : N_of_orbitals ) = basis( 1 : k )
+    
+end if
+
```

Comment:
Diego, 
e se N_of_orbitals for impar?
e o spin, foi duplicado?
Não tenho certeza dessa mudança.